Integration-guides/SSP-Integration-Summary.md

```diff
@@ -0,0 +1,35 @@
+# Integration Summary for SSPs
+SSPs need to integrate with OneKey. OneKey ids is an interoperable ids whose usage is governed by the model terms.
+
+## Model terms
+Details about the terms governing the usage of the id can be found here [Model terms](/./model-terms/model-terms-v1.1.md). 
+Model terms are to be including in bilateral contract with DSPs or any other exchange using the ids in order to use the OneKey framework.
+
+### Noticeable restrictions
+
+* Ids come with preferences that are to be respected.
+* You should only transfer the id and preferences to parties with whom you have a contract including the model terms.
+* OneKey ids should not be associated with offline PII, except for those who collected those PII on their own websites.
```

Comment:
To remove.

- Model Terms don't use the term "PII". It's a term that require careful definition. The most accepted meaning in legal context is often no the one understood in tech docs.
- This sentence is likely a misinterpretation of  6.2
- SSP are anyway unlikely to use the ids and preferences for anything but forwarding it to bidders, so it's not crucial to reiterate here.
- If there are bid request enrichments that would be restricted by the Model Terms, we should come back to this but in another review.

Reply:
Removed for SSP and updated for DSP